web/styles/compose.css

```diff
@@ -55,8 +55,12 @@
             border-radius: 3px;
         }
 
-        #left_bar_compose_reply_button_big {
+        .compose_reply_button_wrapper {
             flex-grow: 1;
+        }
+
+        #left_bar_compose_reply_button_big {
+            width: 100%;
```

Comment:
Ah, I guess we don't lose it completely. I think this `width: 100%` looks to be doing the job that `flex-grow: 1` was previously; maybe replicate that here? Is there a reason to set the width instead of letting flexbox do its thing?

Reply:
Oh, I think I see see. The wrapper makes this no longer a flex item. Is there a way to pull this off without introducing the wrapper element? Or is that the secret-sauce to making sure that the tooltip appears as and when expected?

Reply:
According to the Tippy.js [docs](https://atomiks.github.io/tippyjs/v6/constructor/#disabled-elements), using wrapper elements for disabled elements does seem like the secret-sauce over here.